docs/stats/robust.rst

```diff
@@ -0,0 +1,105 @@
+.. _stats-robust:
+
+******************************
+Robust Statistical Estimators
+******************************
+
+Robust statistics provide reliable estimates of basic statistics for complex
+distributions.  The statistics package includes several robust statistical 
+functions that are commonly used in astronomy.  This includes methods for 
+rejecting outliers as well as statistical description of the underlying 
+distributions.  
+
+In addition to the functions mentioned here, models can be fit with outlier 
+rejection using :func:`~astropy.modeling.fitting.FittingWithOutlierRemoval`.
+
+Sigma Clipping
+==============
+
+Sigma clipping provides a fast method to identify outliers in a distribution.  For a distribution of points, a center and a standard deviation
+are calculated.   Values which are a set number of sigma times the standard deviation away from the center are rejected.  The process can be iterated to further reject outliers.
+
+In the stats package, there is the :func:`~astropy.stats.sigma_clip` function for sigma clipping a distribution.  The function returns a masked array
+where the rejected points are masked.   Sigma clipping can be applied as follows::
```

Comment:
Because this relies on `scipy` (optional dependency), you'll need to add:

`.. doctest-requires:: scipy` before this code block and change the end of the line above from "follows::" to "follows:" (one colon).

Also, we generally prefix inline code with `>>> `.